website/pages/blog/graphql-yoga-v3.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Announding GraphQL Yoga v3
+tags: [graphql, nodejs, typescript, yoga]
+authors: [saihaj, arda, enisdenjo, laurin]
+date: 2022-11-15
+description: The new version of GraphQL Yoga is out! Learn what has improved and what is new!
+image: /x
+thumbnail: /x
+---
+
+After months of experimentation and learning from production use cases we are back again with a ground breaking release for GraphQL Yoga.
+
+## The Yoga v3 Experience
+
+The goal with Yoga is to empower developers, improve the developer facing API and make caching, security and real-time easy and run anywhere.
+Let's highlight some of the groundbreaking new features and improvements.
+
+### One Package to rule them all
+
+From GraphQL Yoga v2 we figure that using the `fetch` API as standard is the way to go.
+Now GraphQL Yoga has a single NPM package for all environments `graphql-yoga` instead of `@graphql yoga/common` and `@graphql-yoga/node`.
```

Comment:
```suggestion
Now GraphQL Yoga has a single NPM package for all environments `graphql-yoga` instead of `@graphql-yoga/common` and `@graphql-yoga/node`.
```